cygnus-ngsi/resources/ngsi-examples/deprecated/ngsiv1-notification-json-compound.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
```

Comment:
This is not a new file, but a moved one. This time it seems github doesn't have detected correctly the move and mark this as a removal + addition in the diff tab of the PR.

NTC (informative)